src/libostree/ostree-repo-commit.c

```diff
@@ -3272,8 +3272,13 @@ get_final_xattrs (OstreeRepo *self, OstreeRepoCommitModifier *modifier, const ch
   if (modifier && modifier->sepolicy)
     {
       g_autofree char *label = NULL;
+      const char *path_for_labeling = relpath;
 
-      if (!ostree_sepolicy_get_label (modifier->sepolicy, relpath,
+      if ((modifier->flags & OSTREE_REPO_COMMIT_MODIFIER_FLAGS_SELINUX_LABEL_V1) > 0
+          && g_str_has_prefix (relpath, "/usr/etc"))
```

Comment:
To be more strict, this should probably be something like

```suggestion
          && (g_str_equal (relpath, "/usr/etc") || g_str_has_prefix (relpath, "/usr/etc/")))
```